### PR TITLE
Initial Setup for Flexview Project

### PR DESCRIPTION
**Pull request by jherwig1**
Configures the build process and sets up the Django server.

### CODE REVIEW

File: denvercode/flexview/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+django<1.10
```
Comment (jduggan1) [inline denvercode/flexview/requirements.txt]: I think we should move to 1.10.

1.9 support ends in April, at which point 1.11 (an LTS release) will be released. We could just jump straight from 1.9 to 1.11, but it seems like that would introduce double the amount of headaches at what is usually a busy time for everyone.

File: denvercode/flexview/package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "bower": "1.7.9",
+    "grunt": "1.0.1",
+    "grunt-contrib-requirejs": "1.0.0",
+    "grunt-contrib-watch": "1.0.0",
+    "grunt-jsdoc": "2.1.0"
+  },
+  "devDependencies": {
+    "jshint": "2.9.2"
+  },
+  "name": "npmap-species",
```
Comment (jduggan1) [inline denvercode/flexview/package.json]: Call it flexview. 😃

File: denvercode/flexview/package.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+{
+  "description": "A Django-based web application for processing and visualizing streaming power system data.",
```
Comment (jduggan1) [inline denvercode/flexview/package.json]: I guess this isn't that important, but we should probably change this as well.

"A Django-based web application for visualizing species' habitats and relationships."?

File: denvercode/flexview/npmap/settings.py [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+
+def from_base_dir(*paths):
+    return os.path.abspath(os.path.join(BASE_DIR, *paths))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.9/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '83vr7q2c!==@uu=oze%6f@q&pp5ifb*hh^hb3j=_altxf5-0lc'
```
Comment (jduggan1) [inline denvercode/flexview/npmap/settings.py]: Is this supposed to be a secret haha?

File: denvercode/flexview/package.json (hunk 3 above)
Comment (jduggan1) [inline denvercode/flexview/package.json]: Actually, just re-use what you wrote for the bower description, "A Django-based web application for visualizing species distributions."

File: denvercode/flexview/bower.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+	"dependencies": {
+		"almond": "0.3.2",
+		"jquery": "2.2.4",
+		"leaflet": "1.0.0-rc.2"
+	},
+	"description": "A Django-based web application for visualizing species distributions.",
+	"name": "npmap"
```
Comment (jduggan1) [inline denvercode/flexview/bower.json]: flexview

**Comment by jduggan1** [comment]
I've added a few minor comments, but the only comment of real substance is the suggestion to move to Django 1.10. Could you give it a try and see if it will be an easy change to make (I'm thinking at this stage that the answer will be yes)?

File: denvercode/flexview/npmap/settings.py (hunk 4 above)
Comment (jherwig1) [inline denvercode/flexview/npmap/settings.py]: lol I guess. The secret key was in the curent repo as well.

**Pull request merged**